ceph-pr-submodules/config/definitions/ceph-pr-commits.yml

```diff
@@ -0,0 +1,54 @@
+- job:
+    name: ceph-pr-submodules
```

Comment:
Do you want to lock this to a certain node type? I think without this defined it'll use master which I don't think we want.